pkg/pod/ttlDestroyHandler.go

```diff
@@ -28,11 +34,20 @@ func (t ttlDestroyHandler) Supports() string {
 func (t ttlDestroyHandler) Handle(pod v1.Pod) error {
 	log.Printf("TTL handler invoked: %s", pod.Name)
 
-	creationTimestamp := pod.Labels["im-creation-timestamp"]
-	ttl := pod.Labels["im-ttl"]
-	if creationTimestamp == "" || ttl == "" {
-		log.Println("No creationTimestamp or TTL found")
-		return nil
+	creationTimestampLabel := pod.Labels["im-creation-timestamp"]
+	ttlLabel := pod.Labels["im-ttl"]
+	if creationTimestampLabel == "" || ttlLabel == "" {
+		return errors.New("no creationTimestamp or TTL found")
```

Comment:
This was an error before, just wondered why. It does not seem like an error just if there is nothing to do 🤔

Reply:
That's a blunder on my behalf. Since we're now setting the creation label on all instances I guess it not being there should result in an error. While the lack of TTL label should just result in continuing. I've updated the code to reflect this.